exporter/logzioexporter/internal/dbmodel/logziospan.go

```diff
@@ -0,0 +1,66 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
why is exporter/logzioexporter/logziospan.go file shown as deleted instead of moved?